### PR TITLE
feat(website/home): update home page

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [x] No

## Describe the changes

Update content of the home page. Add like for Contributing and Patterns Status page.

Github issue number or Jira issue URL: **MZC-216**

### CODE REVIEW

**Comment by nherwegh** [comment]
I think we're missing some capitalisation in the head of the table (pattern, vue, react, angular)

**Comment by nherwegh** [comment]
@tiloyi for some inspiration : 
[Lexicon](https://liferay.design/lexicon/news/status/)

**Comment by gael-boyenval** [comment]
@nherwegh : I agree with you on all the things you pointed but this PR is only meant to links to existing pages of the website on the home page. 
To create a new feature or improve the existing pages, create a ticket.
As a full member of this team you gained the right to make awesome designs for the styleguide and put stars in hour eyes 🤩.

**Comment by nherwegh** [comment]
> can't we try to give more informations to the user instead of a big unreadable table ?
> 
> 💡categorise elements ?
> 💡give the user a legend of what status means what ?

not relevant as they say

File: src/pages/index.js [hunk 1]
```diff
@@ -213,6 +219,31 @@ const IndexPage = () => (
...
+          <Tile
+            bottom
+            center
+            title="Contributing"
+            text="Learn how to contribute to Mozaic, report a bug, submit changes or ask help"
+            linkurl="/Contributing/"
+            linktitle="Contribute to Mozaic"
+            link="Discover"
+          ></Tile>
+          <Tile
```
[the remark this answers is not in the document]
Comment (nherwegh) [reply src/pages/index.js]: maybe we can put this `pattern status` in a section like changelog or something like that ?

File: src/pages/index.js [hunk 2]
```diff
@@ -213,6 +219,31 @@ const IndexPage = () => (
...
+            bottom
+            center
+            title="Patterns Status"
+            text="See in real time the status of Mozaic patterns."
```
Comment (nherwegh) [inline src/pages/index.js]: we should say: 
See Mozaic patterns status in real time.

File: src/pages/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (gael-boyenval) [reply src/pages/index.js]: Yes I think the page will ultimately go elsewhere I was thinking something like: 

- updates
  - patterns status
  - changelog
  - roadmap

[the remark this answers is not in the document]
Comment (tiloyi) [reply src/pages/index.js]: @gael-boyenval i need more precision about your last comment. Where do you want me to put this information? In the homepage ?

File: src/pages/index.js [hunk 3]
```diff
@@ -213,6 +219,31 @@ const IndexPage = () => (
...
       </Section>
     </Slice>
+    <Slice large>
+      <Section title="Contribute">
+        <div className="ml-flexy ml-flexy--items-stretch ml-flexy--gutter">
+          <Tile
+            bottom
+            center
+            title="Contributing"
+            text="Learn how to contribute to Mozaic, report a bug, submit changes or ask help"
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/pages/index.js]: fixed.

File: src/pages/index.js [hunk 4]
```diff
@@ -213,6 +219,31 @@ const IndexPage = () => (
         </div>
       </Section>
     </Slice>
+    <Slice large>
+      <Section title="Contribute">
```
[the remark this answers is not in the document]
Comment (tiloyi) [reply src/pages/index.js]: fixed

File: src/pages/index.js (hunk 2 above)
Comment (tiloyi) [reply src/pages/index.js]: @nherwegh as said in a previous feeback upper, i removed the "Pattern status" section.

**Pull request merged**